lib/puppet/module_tool/errors/installer.rb

```diff
@@ -50,7 +59,8 @@ def initialize(original, options)
     end
 
     def multiline
-      _(<<-MSG).strip % { module_name: @requested_module, version: @requested_version, dir: @directory }
+      # TRANSLATORS "mkdir -p'%{dir}'" is a command line example and should not be translated
```

Comment:
Is this the way we ensure command line invocations aren't translated? I thought in clojure we put the command line into a variable to be interpolated in the string (but that could be old or inaccurate).

Reply:
I was just following the rest of what I was seeing, which is to put a comment for the translators

Reply:
I went back through and looked at some clojure projects and it looks like we're not at all consistent in it, so I must be mis-remembering. From confluence it looks like the translator comment is the best practice: https://confluence.puppetlabs.com/display/enterprise/Translation-friendly+strings#Translation-friendlystrings-Commandsandtechnicalterminology